examples/mediatek/executor_runner/mtk_llama_runner.cpp

```diff
@@ -7,16 +7,6 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-/* Copyright Statement:
- *
- * This software/firmware and related documentation ("MediaTek Software") are
- * protected under relevant copyright laws. The information contained herein
- * is confidential and proprietary to MediaTek Inc. and/or its licensors.
```

Comment:
This shouldn't be proprietery to MediaTek. According to CLA, the code is open sourced with BSD license.